service/src/main/java/bio/terra/workspace/service/policy/TpsRetry.java

```diff
@@ -100,6 +101,9 @@ private <T> T perform(TpsRetry.TpsFunction<T> function)
         } else {
           throw ex;
         }
+      } catch (ProcessingException ws) {
```

Comment:
Confirming my understanding -- all `ProcessingExceptions` are retryable (unlike `ApiExceptions`, only a subset of which are retryable).